service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/CaseFlagControllerSubmittedTest.java

```diff
@@ -72,7 +73,7 @@ public void shouldUpdateSummaryCaseFields() {
 
         postSubmittedEvent(toCallBackRequest(caseData, caseDataBefore));
 
-        verify(coreCaseDataService).performPostSubmitCallback(
+        verify(coreCaseDataService, timeout(10000)).performPostSubmitCallback(
             eq(caseData.getId()),
```

Comment:
@AlistairEdwinOsborne I see you added ASYNC_METHOD_CALL_TIMEOUT to the abstract class. Just checking you didn't mean to refer to that, instead of 10000, here.